core/src/main/java/org/bitcoinj/wallet/AuthenticationKeyChainFactory.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2014 devrandom
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Incorrect copyright notice?

Reply:
probably, this was from copy/pasting from a different file

Reply:
https://github.com/dashevo/dashj/pull/49/commits/a2b9daa4777312b1f11d05cc2a984a993179ebe6